Tests/CryptoTests/Utils/XCTestUtils.swift

```diff
@@ -0,0 +1,61 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCrypto open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftCrypto project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of SwiftCrypto project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+
+// Xcode 11.4 catches errors thrown during tests and reports them on the
+// correct line. But Linux and older Xcodes do not, so we need to use this
+// wrapper as long as those platforms are supported.
+func orFail<T>(file: StaticString = #file, line: UInt = #line, _ closure: () throws -> T) throws -> T {
+    func wrapper<T>(_ closure: () throws -> T, file: StaticString, line: UInt) throws -> T {
+        do {
+            return try closure()
+        } catch {
+            XCTFail("Function threw error: \(error)", file: file, line: line)
+            throw error
+        }
+    }
+
+    #if compiler(>=5.2) && !os(Linux)
```

Comment:
```suggestion
    #if compiler(>=5.2) && canImport(Darwin)
```

This would be more general and better convey the intent, since there's nothing Linux-specific here.